client/src/App.js

```diff
@@ -1,12 +1,12 @@
 import { Route, Routes } from "react-router-dom";
 
 import About from "./pages/About";
-import Home from "./pages/Home";
+import LoginPage from "./pages/LoginPage";
 
 const App = () => (
 	<Routes>
-		<Route path="/" element={<Home />} />
-		<Route path="/about/this/site" element={<About />} />
+		<Route path="/" element={<LoginPage />} />
+		<Route path="/about/this/side" element={<About />} />
```

Comment:
did youo mean to put 'site' here?

Reply:
This will be updated by John as discussed. It will be the path for StudentPage

Reply:
ah no worries then. once John has reviewed the branch can be merged and I'll pull this and fix us up a login component as well as some nice buttons with material UI, I've finished watching a video on it now and I think I get the jist - I also have the docs open in a tab ready too :D